common/workunit/workunit.cpp

```diff
@@ -3659,6 +3659,21 @@ class CLightweightWorkunitInfo : public CInterfaceOf<IConstWorkUnitInfo>
         appvalues.loadBranch(&p,"Application");
         totalThorTime = (unsigned)nanoToMilli(extractTimeCollatable(p.queryProp("@totalThorTime"), nullptr));
         _isProtected = p.getPropBool("@protected", false);
+
+        Owned<IPropertyTreeIterator> iterCostExecute = p.getElements("./Statistics/Statistic[@kind='CostExecute'][@scope='']");
```

Comment:
design: I wonder if it should really be stored as @executedCost in the workunit when it is aggregated.  It would add a minor impact when aggregating, but would save a relatively complex data transfer when getting the workunit information.
The other issue is how does this work with cassandra workunits?  If you stored it in an attribute I think it would be a relatively trivial change.  Without it it may be complex.
You could take it further and create an implicit statistic from the workunit property, but that probably doesn't have a major benefit.